action/protocol/staking/vote_bucket.go

```diff
@@ -294,6 +294,14 @@ func getAllBuckets(sr protocol.StateReader) ([]*VoteBucket, uint64, error) {
 	return buckets, height, nil
 }
 
+func getStakingBuckets(sr protocol.StateReader) (ret *iotextypes.VoteBucketList, err error) {
```

Comment:
feel we don't need this as an extra func
just directly write the 5 lines in handleStakingIndexer()